nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -1483,10 +1483,14 @@ def build_transformer_config(self) -> TransformerConfig:
         activation_func = activation_to_func(activation)
 
         normalization = self.cfg.get('normalization', 'layernorm')
+        layernorm_zero_centered_gamma = self.cfg.get('layernorm_zero_centered_gamma', 'False')
```

Comment:
see above comment. Can set this to be False instead

Reply:
can you change this line to just `layernorm_zero_centered_gamma = False` ?

Reply:
@gshennvm I think it's better lo leave it as it is since we have ` layernorm_zero_centered_gamma` param in Launcher configs for improved models. What do you think?

Reply:
I see, ok with me. Users can customize it if needed

Reply:
@MaximumEntropy given that we're moving to mcore and the user is likely not going to be exposed to the layernorm1p argument if they are coming from mcore. I was thinking maybe we can have both options. (i.e. if the user is more used to mcore they can say LayerNorm + layernorm_zero_centered_gamma). wdyt?